Runtime/Utils/Values/SubscribableDictionaryValue.cs

```diff
@@ -0,0 +1,182 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Padoru.Core
+{
+    public enum DictionaryEvent
+    {
+        DictionaryChanged = 0,
+        KeyAdded = 1,
+        KeyRemoved = 2,
+        KeyValueChanged = 3,
+    }
+    
+    [Serializable]
+    public class SubscribableDictionaryValue<TKey, TValue> : IDictionary<TKey, TValue>
```

Comment:
Rename the class to `SubsctibableDictionary`. Essentially remove the `Value` part since it is redundant in this case. Do the same with the `SubscribableListValue` class